PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingDetailVC.swift

```diff
@@ -0,0 +1,271 @@
+//
+//  PetListingDetailVC.swift
+//  PetAdoption-iOS
+//
+//  Created by Keli'i Martin on 8/25/16.
+//  Copyright © 2016 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+import PetAdoptionTransportKit
+
+enum Sections: Int
+{
+    case description = 0
```

Comment:
explicit numbering is not required when starting at 0. 

not worth updating, just a nice to know.